ui_test/src/lib.rs

```diff
@@ -426,7 +409,28 @@ fn check_annotations(
             lowest_annotation_level = std::cmp::min(lowest_annotation_level, level);
         }
 
-        if let Some(msgs) = messages.get_mut(line) {
+        let (line, definition_line) = match lines {
```

Comment:
Can you factor this logic of "does an `ErrorLines` match this line" into a separate function?